src/app/api/routes/devices.py

```diff
@@ -1,60 +1,82 @@
 from typing import List
-from fastapi import APIRouter, Path, Security
+from datetime import datetime
+from fastapi import APIRouter, Path, Security, HTTPException
 
-from app.api import routing
+from app.api import crud
 from app.db import devices
-from app.api.schemas import DeviceOut, DeviceAuth, DeviceIn, UserRead, HeartbeatOut, DefaultPosition, Cred
+from app.api.schemas import DeviceOut, DeviceAuth, DeviceCreation, DeviceIn, UserRead, DefaultPosition, Cred
 from app.api.deps import get_current_device, get_current_user
 
+from app.api.routes.accesses import post_access, update_access_pwd
+
 router = APIRouter()
 
 
 @router.post("/", response_model=DeviceOut, status_code=201)
 async def create_device(payload: DeviceAuth, _=Security(get_current_user, scopes=["admin"])):
-    return await routing.create_device(devices, payload)
+    access_entry = await post_access(payload.name, payload.password, scopes=payload.scopes)
```

Comment:
I used to agree, but I was thinking about best practices in django and other habits. However, here fastapi was created, among others, to overcome the limitations of other frameworks. The use of the router decorator does not suggest that it has to be a one liner cf. https://fastapi.tiangolo.com/tutorial/bigger-applications/#another-module-with-apirouter

What I did in the access route is only because both functions are called by different routes. For devices, and users, I no longer see the point of creating another function just to wrap it with the decorator. I only this as a slight computation overhead for passing this down to another function.

Writing other functions, at least for me on this project, should be made to avoid duplication when this operation is done several times in the codebase. But I'm biased here, what are the reasons you think we should wrap this?